db/schema.rb

```diff
@@ -153,6 +153,7 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.boolean  "superapp",     default: false, null: false
+    t.string   "website"
```

Comment:
I originally didn't go with URL because I didn't want it muddled with the redirect uri/url. If that's not likely, then sure, if not, website_url works.